doc/whats_new.rst

```diff
@@ -44,7 +44,7 @@ Detailed list of changes
 - Add :meth:`mne_bids.BIDSPath.find_matching_sidecar` to find the sidecar file associated with a given file path by `Eric Larson`_ (:gh:`1093`)
 - When writing data via :func:`~mne_bids.write_raw_bids`, it is now possible to specify a custom mapping of :class:`mne.Annotations` descriptions to event codes via the ``event_id`` parameter. Previously, passing this parameter would always require to also pass ``events``, and using a custom event code mapping for annotations was impossible, by `Richard Höchenberger`_ (:gh:`1084`)
 - Improve error message when :obj:`~mne_bids.BIDSPath.fpath` cannot be uniquely resolved by `Eric Larson`_ (:gh:`1097`)
-- Add :func:`mne_bids.find_matching_paths` to get all bids paths that match the requirements by :newcontrib:`Moritz Gerster` (:gh:`1103`)
+- Add :func:`mne_bids.find_matching_paths` to get all bids paths that match the requirements by `Moritz Gerster`_ (:gh:`1103`)
```

Comment:
I worked a bit on the phrasing; this is just a suggestion:
```suggestion
- Add :func:`mne_bids.find_matching_paths` to retrieve all `BIDSPaths` matching user-specified entities. The functionality partially overlaps with what's offered through `mne_bids.BIDSPath.match()`, but is more versatile, by `Moritz Gerster`_ (:gh:`1103`)
```

Reply:
I like it @hoechenberger! Your version is more precise 👍

Reply:
@moritz-gerster you can click the "commit suggestion" button to incorporate it into your code. Just make sure that: 1) you don't have uncommitted changes locally, and 2) you `git pull` before continuing to work locally